build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.2.41'
+    ext.kotlin_version = '1.2.51'
```

Comment:
Should we use a variable to make the kotlin version and findbugs version easier to keep up to date throughout the app.